src/context-parser.js

```diff
@@ -297,6 +308,8 @@ FastParser.prototype.lookupChar = function(ch) {
 
 /**
  * @function FastParser#contextualize
+ *
+ * @param {string} input - the input stream
```

Comment:
btw, the prototype of `FastParser.contextualize()` is different from `Parser.contextualize()`. the one with Parser returns the modified (canonicalized) input, whereas that of FastParser never returns.
c.c. @neraliu